tests/unittests/tests-hacl/Makefile.include

```diff
@@ -0,0 +1,4 @@
+USEMODULE += random
+USEPKG += hacl
+
+CFLAGS += -DTHREAD_STACKSIZE_MAIN=\(2*THREAD_STACKSIZE_DEFAULT+THREAD_EXTRA_STACKSIZE_PRINTF\)
```

Comment:
It is better to add your unittest to the `LARGE_STACK_TESTS at l217 of tests/unittests/Makefile, instead of adding the cflag here.

And then add a note about stack requirements to the package doc.txt (same way as tweetNaCl)

Reply:
+1 remove this line

Reply:
and add LARGE_STACK_TESTS += tests-hacl in unittests/Makefile